test/component_test.go

```
@@ -0,0 +1,525 @@
+	}
+
+	output, err := client.CreateRegexPatternSet(context.Background(), regexSetInput)
+	if err != nil {
+		return nil, err
+	}
+
+	return output.Summary, nil
+}
+
+func getIPSetByARN(t *testing.T, client *wafv2.Client, arn string) *wafv2.GetIPSetOutput {
+	ipSets, err := client.ListIPSets(context.Background(), &wafv2.ListIPSetsInput{
```

golangci-lint (1.62.2)

ineffectual assignment to err

(ineffassign)